src/components/SiteDetailSubItems.js

```diff
@@ -1,19 +1,33 @@
 import React from 'react';
 
 import styled from 'styled-components/macro';
-import { ThemeProvider } from 'styled-components/macro';
-import { ButtonStyle, MenuLink } from '../styles/MermaidStyledComponents';
 import { theme } from '../constants/theme';
+import { withStyles } from '@material-ui/core/styles';
 import ContactIcon from '@material-ui/icons/Email';
 
+import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
+import DownloadDataModal from './DownloadDataModal';
+import { useMediaQuery } from '@material-ui/core';
+import TooltipLayout from './TooltipLayout';
```

Comment:
Im not sure about the name `TooltipLayout` its not really a layout component, just a custom tooltip. Could it be called `MermaidTooltip` or `MermaidTooltip1` if there are others, or `MermaidDashboardTooltip` if its different than the collecting app?.